romancal/lib/dms.py

```diff
@@ -0,0 +1,42 @@
+"""Functionality required by DMS outside of core functionality of the package
+
+Example: Certain tests are required by DMS to log in a specific way. The
+decorator for this is defined in this module.
+"""
+from functools import wraps
+
+from metrics_logger.decorators import metrics_logger
+from stpipe import log as stpipe_log
+
+
+def result_logger(requirement, message):
```

Comment:
I thought the metrics_logger package already did this for us, and we shouldn't need to add anything?

Reply:
This was what generated my question on slack about the double-logging, which is seen in other tests. If there is no particular need for this, I am happy to pull this out and rely simply on the provided metrics logger.

Reply:
Let's use the metrics_logger as in the other tests.